### PR TITLE
Remove store limit scene

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Remove the store limit scene feature since it isn't maintained and well tested.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by en-jin19 — APPROVED** [review]
LGTM

**Review by lhy1024 — APPROVED** [review]

File: pd-control.md [hunk 1]
```diff
@@ -148,21 +148,14 @@ export PD_ADDR=http://127.0.0.1:2379 &&
 ```
 {
   "replication": {
-    "enable-placement-rules": "false",
+    "enable-placement-rules": "true",
```
Comment (TomShawn) [inline pd-control.md:151]: 从 v5.0 后，Placement Rules 默认开启。这个 PR 是否需要 cherr-pick 自 v5.0、v5.1 和 v5.2 呢？

Comment (en-jin19) [reply pd-control.md:151]: 感谢提醒！刚刚确认到，我们可以先 cherry pick 到 4.0~5.3 版本，其中，在 5.X 版本中修改 Placement Rules 部分，在所有版本中删除 limit-scene 部分。

**Review by TomShawn — APPROVED** [review]

**Pull request merged**